packages/frontend/core/src/mixpanel/events.ts

```diff
@@ -0,0 +1,153 @@
+// let '$' stands for unspecific matrix
+/* eslint-disable rxjs/finnish */
+export interface Events {
```

Comment:
IMO it seems it is a bit hard to figure out the relationship between layers to module/page/segment etc.
Not sure if it is better/possible to divide this monolith definition into pieces?

Reply:
I tried but it only went more verbose. I will make it clear regarding the layers after tests